rust/operator-binary/src/kafka_controller.rs

```diff
@@ -293,6 +293,27 @@ pub enum Error {
     GracefulShutdown {
         source: crate::operations::graceful_shutdown::Error,
     },
+
+    #[snafu(display("failed to get required labels"))]
+    GetRequiredLabels {
+        source:
+            stackable_operator::kvp::KeyValuePairError<stackable_operator::kvp::LabelValueError>,
+    },
+
+    #[snafu(display("failed to build metadata"))]
+    MetadataBuild {
+        source: stackable_operator::builder::ObjectMetaBuilderError,
+    },
+
+    #[snafu(display("failed to build labels"))]
+    LabelBuild {
+        source: stackable_operator::kvp::LabelError,
+    },
+
+    #[snafu(display("failed to add Secret Volumes and VolumeMounts"))]
+    AddVolumesAndVolumeMounts {
+        source: stackable_kafka_crd::security::Error,
+    },
```

Comment:
I dont find that too consistent. "labels" and "metadata" lower case (both Kubernetes machinery), Volume and VolumeMount capitalized / camelCase. Whats the reasoning here?

Reply:
https://github.com/stackabletech/kafka-operator/pull/653/commits/4399c8baed769cba615e6ec589fc5a2dc1ef20c2